index.yaml

```diff
@@ -18,3 +18,9 @@ plugins:
       maintainer: aquasecurity
       summary: Put referrers to OCI registry
       output: false
+    - name: aws-scan
+      version: 0.1.0
+      repository: github.com/aquasecurity/trivy-aws
+      maintainer: aquasecurity
+      summary: AWS Cloud Scanning
+      output: false
```

Comment:
This is auto-generated. I'll add a comment on the top like `// DO NOT EDIT: generated by a script`.

Reply:
Added https://github.com/aquasecurity/trivy-plugin-index/pull/8/commits/6dbac907f6fea46251e98981ecc3a9cc3c90af10

Reply:
Thanks! Cherry-picked.
https://github.com/aquasecurity/trivy-plugin-index/pull/9